extras/parse_travis_log.py

```diff
@@ -388,7 +388,9 @@ def makebuild_summary(log_filename, msg_make_section_start,
                     error_summary[file_name] += 1
                     number_of_error_msgs += 1
 
-                if ': warning:' in line:
+                # Only count warnings originating in NEST source files
+                m = re.match("/home/travis/build/nest/nest-simulator.*: warning:", line)
```

Comment:
In the worst case (full build with zillions of warnings), we perform this match some 20000 times. Would it make sense to compile the expression outside the loop, something like
```python
nest_warning = re.compile('/home/travis/build/nest/nest-simulator.*: warning:')
...
if nest_warning.match(line):
   ...
```

Reply:
Good idea, thanks!

Reply:
Done in 54893be5d4231e7b7c53084a242c07ee3ff6418d.